ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/converters/ConvertType.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2015 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3autogen.java.converters;
+
+public class ConvertType {
+
+    private final static String MODELS_PACKAGE = "com.spectralogic.ds3client.models.";
```

Comment:
Should we move all these constant fields like this into a common Constants class so we can see what all the package names are in a single file?